digital_asset_types/src/dao/mod.rs

```diff
@@ -198,6 +192,23 @@ impl SearchAssetsQuery {
             joins.push(rel);
         }
 
+        if let Some(cv) = self.creator_verified {
```

Comment:
I think this could might be a bit cleaner if you add the join in one line, when either condition exists, and then add each condition separately. feels easier to read to me. you should also add a comment to clarify the behaviour